autoprotocol/protocol.py

```diff
@@ -5047,9 +5047,9 @@ def provision(self, resource_id, dests, volumes):
         dests : Well or WellGroup or list(Well)
           Destination(s) for specified resource.
         volumes : str or Unit or list(str) or list(Unit)
```

Comment:
I understand there was the comment with @joshhacksthings on this... to use `volumes` rather than `amounts` for backwards compat. 

However, having `volumes` as the docstring/argument name seems pretty unintuitive. One possible workaround to support backwards compat would be to add volumes as a kwarg (keyword arugment).
I.e. `provision(self, resource_id, dests, amounts, volumes=None)`

Hmm.. for now we can add a comment that it's for backwards compat/will be deprecated. Forward thinking, we could look into something similar to https://pypi.org/project/Deprecated/ as well. That would be nice for marking these.

What do you think, @joshhacksthings ?